docs/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments.md

```diff
@@ -18,14 +18,14 @@ The C# compiler processes documentation comments in your code and formats them a
   
 ## Tags  
   
-||||  
-|---|---|---|  
-|[\<c>](./code-inline.md)|[\<para>](./para.md)|[\<see>](./see.md)*|  
+|||||  
+|---|---|---|---|
+|[\<c>](./code-inline.md)|[\<para>](./para.md)|[\<see>](./see.md)*|[\<value>](./value.md)  
 |[\<code>](./code.md)|[\<param>](./param.md)*|[\<seealso>](./seealso.md)*|  
 |[\<example>](./example.md)|[\<paramref>](./paramref.md)|[\<summary>](./summary.md)|  
 |[\<exception>](./exception.md)*|[\<permission>](./permission.md)*|[\<typeparam>](./typeparam.md)*|  
 |[\<include>](./include.md)*|[\<remarks>](./remarks.md)|[\<typeparamref>](./typeparamref.md)|  
-|[\<list>](./list.md)|[\<returns>](./returns.md)|[\<value>](./value.md)|  
+|[\<list>](./list.md)|[\<inheritdoc>](./inheritdoc.md)|[\<returns>](./returns.md)|
```

Comment:
It was ordered alphabetically. Now It's not. The new inheritdoc should come just after the include one.